### PR TITLE
Adding store state helpers

### PR DESCRIPTION
**Pull request by kenwheeler**
@eastridge Added the following store state helpers:

``` js
Store.getPending() // Returns store's pending state
Store.setPending(arg) // Sets store's pending state
Store.getErrors() // Returns store's errors array
Store.setError(arg) // Pushes an error onto the store's errors array
Store.clearErrors() // Clears the store's errors array
```

### CODE REVIEW

**Comment by eastridge** [comment]
Should `setPending`, `setError` and `clearErrors` all be pseudo private underscore prefixed methods?

**Comment by kenwheeler** [comment]
Yeah, for some reason I thought they might be called from anywhere other than the payload callback. Updating now.

**Comment by kenwheeler** [comment]
Any reason to go pseudo private vs full private?

**Comment by eastridge** [comment]
If it's full private you have to write it yourself right?

**Comment by kenwheeler** [comment]
You would have to anyway right? Like you would setPending(true) in response to an 'ACTION_START', and then setPending(false) on 'ACTION_SUCCESS' or 'ACTION_ERROR'

**Comment by eastridge** [comment]
No I mean, if they were truly private you'd have to have a local `setPending` function you implement yourself for each and every store vs a `_setPending` function you already have on the instance.

**Comment by kenwheeler** [comment]
Yes it would be the latter

**Comment by kenwheeler** [comment]
@eastridge Weirdest thing, when I underscore prefix those methods, they aren't available in the tests. I think it might be a Jest thing.

**Comment by kenwheeler** [comment]
It works in /dist and /lib , looks like Jest ignores underscore prefixed methods for some reason. Took those tests out, but made sure it worked via link. Good to go?

File: lib/Store.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ var Store = (function () {
 
     var self = this;
     this.callback = callback;
+    this.pending = false;
```
Comment (rgerstenberger) [inline lib/Store.js]: These will always be internal to Biff right? Should they be something like `__pending` so there aren't strange collisions?

Comment (kenwheeler) [inline lib/Store.js]: Fair point. I'll underscore them.

**Comment by eastridge** [comment]
Are the double underscores to avoid jest issues? Normally we go with single, otherwise LGTM. Generally I do not want to be a blocker on PRs, so don't consider me a blocker on anything unless I've specifically instructed otherwise

**Comment by kenwheeler** [comment]
Nope, I'll switch them to single and merge, I thought it was an internal js style thing.

**Pull request merged**